source/performance/F0.CodeAnalysis.Benchmarking/CodeAnalysis/DiagnosticAssert.cs

```diff
@@ -108,6 +110,7 @@ private static void AreEqual(Diagnostic expectedDiagnostic, Diagnostic actualDia
 		}
 	}
 
+	[SuppressMessage("Globalization", "CA1305:Specify IFormatProvider", Justification = ".NET Standard 2.0")]
```

Comment:
We're getting some new warnings suggesting to use method overloads from greater TFMs due to multi-targeting _.NET 7.0_ in two projects that previously targeted _.NET Standard 2.0_ only.
```diff
-<TargetFramework>netstandard2.0</TargetFramework>
+<TargetFrameworks>net7.0;netstandard2.0</TargetFrameworks>
```